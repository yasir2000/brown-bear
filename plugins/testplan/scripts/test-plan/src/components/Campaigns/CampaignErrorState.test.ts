/*
 * Copyright (c) Enalean, 2020 - present. All Rights Reserved.
 *
 * This file is a part of Tuleap.
 *
 * Tuleap is free software; you can redistribute it and/or modify
 * it under the terms of the GNU General Public License as published by
 * the Free Software Foundation; either version 2 of the License, or
 * (at your option) any later version.
 *
 * Tuleap is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Tuleap. If not, see <http://www.gnu.org/licenses/>.
 */

import { shallowMount } from "@vue/test-utils";
import CampaignErrorState from "./CampaignErrorState.vue";
import { createTestPlanLocalVue } from "../../helpers/local-vue-for-test";

describe("CampaignErrorState", () => {
    it("Displays empty state", async () => {
        const wrapper = shallowMount(CampaignErrorState, {
            localVue: await createTestPlanLocalVue(),
        });

        expect(wrapper.element).toMatchSnapshot();
    });
});
