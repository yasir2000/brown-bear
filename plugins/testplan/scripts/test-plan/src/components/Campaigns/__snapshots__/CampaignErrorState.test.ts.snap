// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CampaignErrorState Displays empty state 1`] = `
<section
  class="empty-state-pane"
>
  <div
    class="empty-state-illustration"
  >
    <campaign-error-state-svg-stub />
  </div>
   
  <div
    class="empty-state-text"
  >
    <translate-stub
      tag="span"
    >
      An error occurred while loading the campaigns.
    </translate-stub>
  </div>
</section>
`;
